phpstan.neon

```diff
@@ -16,3 +17,31 @@ parameters:
         -
             message: '/^Unsafe usage of new static\(\)\.$/'
             path: src/Config.php
+        -
+            message: '/^Else branch is unreachable because previous condition is always true\.$/'
```

Comment:
Maybe use a baseline.neon file tbh?

Reply:
Baseline file is for errors we can't fix right away because of e.g. lack of time, they will be fixed later. Here I don't want to change the code that triggers this error because it is part of public API so I ignore this error on purpose.